### DISCUSSION TITLE
trinity error (the sequel)

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#119] [Q&A] [answered]
Another error: `Error, cannot locate samples file...` 
<img width="1055" alt="Screen Shot 2021-03-04 at 7 52 29 PM" src="https://user-images.githubusercontent.com/66333757/110065032-2a2f4380-7d23-11eb-86ef-fb6775a9564c.png">

Any ideas?

**Comment by sr320** [comment, marked as answer]
Maybe a problem with a space in the drive name. Try moving sample file somewhere else and see if you get same error.

**Reply by meganewing** [reply]
That worked, thanks!
